frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -55,8 +55,13 @@ const handleError = (
   const errorCode = error?.response?.data?.errorCode;
   const unAuthorizedErrorCode = error?.response?.status === ServerCodes.Unauthorized;
 
+  const LogOutUser = () => {
+    logout().then(() => window.location.assign(userLogoutAfterPath()));
+  };
+
```

Comment:
```suggestion
  const LogOutUser = () => logout().then(() => window.location.assign(userLogoutAfterPath()));
```
I also believe we can do like this?

Reply:
fixed